app/about/components/ScrollRevealContainer.tsx

```diff
@@ -5,14 +5,18 @@ import { useRef } from 'react'
 
 export default function ScrollRevealContainer({ children, delay = 0 }) {
   const ref = useRef(null)
-  const isInView = useInView(ref, { once: true, amount: 0.3 })
+  const isInView = useInView(ref, { once: true, amount: 0.1 })
 
   return (
     <div ref={ref}>
       <motion.div
-        initial={{ opacity: 0, y: 40 }}
-        animate={isInView ? { opacity: 1, y: 0 } : { opacity: 0, y: 40 }}
-        transition={{ duration: 0.4, delay }}
+        initial={{ opacity: 0, y: 20 }}
```

Comment:
```suggestion
        initial={{ opacity: 0, y: 20 }}
        whileInView={{ opacity: 1, y: 0 }}
```

whileInView를 쓴다면 ref가 필요없지 않을까?